CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ please take a look at related PRs and issues and see if the change affects you.
 
 ### Fixed
 
+- Backslash escaped chars in string matches ([#323]). Possible **(BIC)**.
```

Comment:
Possible **(BIC)** (escaped chars were not interpreted in string matches prior to this fix).